doc/src/modules/physics/control/control_problems.rst

```diff
@@ -0,0 +1,340 @@
+
+        $$P(s) = \begin{bmatrix}
+        \frac{1}{s} & \frac{2}{s+2} \\
+        0 & 3
+        \end{bmatrix}$$
+
+        $$C(s) = \begin{bmatrix}
+        1 & 1 \\
+        2 & 2
+        \end{bmatrix}$$
+
+2. Also, find the **equivalent closed-loop system** *(or the ratio v/u from the block diagram given below)* for the system (negative-feedback loop) having **C(s)** as the **controller** and **P(s)** as **plant** *(Refer to the block diagram given below)*.
+
+        .. image:: https://user-images.githubusercontent.com/53227127/120820301-0b368c80-c572-11eb-84c4-e372190cf0bd.png
+
+Solution-
+
+    >>> # Imports
+    >>> from sympy import Matrix, pprint
+    >>> from sympy.abc import s, t
+    >>> from sympy.physics.control import *
+
+    Subpart 1
+
+    >>> P_mat = Matrix([[1/s, 2/(2+s)], [0, 3]])
+    >>> C_mat = Matrix([[1, 1], [2, 2]])
+    >>> P = TransferFunctionMatrix.from_Matrix(P_mat, var=s)
+    >>> C = TransferFunctionMatrix.from_Matrix(C_mat, var=s)
+    >>> # Series equivalent, considering (Input)→[P]→[C]→(Output). Note that order of matrix multiplication is opposite to the order in which the elements are arranged.
+    >>> pprint(C*P)
+    [1  1]    [1    2  ]
+    [-  -]    [-  -----]
+    [1  1]    [s  s + 2]
+    [    ]   *[        ]
+    [2  2]    [0    3  ]
+    [-  -]    [-    -  ]
+    [1  1]{t} [1    1  ]{t}
+    >>> # Series equivalent, considering (Input)→[C]→[P]→(Output).
+    >>> pprint(P*C)
+    [1    2  ]    [1  1]
+    [-  -----]    [-  -]
+    [s  s + 2]    [1  1]
+    [        ]   *[    ]
+    [0    3  ]    [2  2]
+    [-    -  ]    [-  -]
+    [1    1  ]{t} [1  1]{t}
+    >>> pprint((C*P).doit())
+    [1  3*s + 8 ]
+    [-  ------- ]
+    [s   s + 2  ]
+    [           ]
+    [2  6*s + 16]
+    [-  --------]
+    [s   s + 2  ]{t}
+    >>> pprint((P*C).doit())
+    [ 5*s + 2    5*s + 2 ]
+    [---------  ---------]
+    [s*(s + 2)  s*(s + 2)]
+    [                    ]
+    [    6          6    ]
+    [    -          -    ]
+    [    1          1    ]{t}
+
+    Subpart 2
+
+    >>> tfm_feedback = MIMOFeedback(P, C, sign=-1)
+    >>> pprint(tfm_feedback.doit())  # ((I + P*C)**-1)*P
+    [    7*s + 14          -s - 6    ]
+    [---------------  ---------------]
+    [   2                2           ]
+    [7*s  + 19*s + 2  7*s  + 19*s + 2]
+    [                                ]
+    [                     2          ]
+    [   -6*s - 12      3*s  + 9*s + 6]
+    [---------------  ---------------]
+    [   2                2           ]
+    [7*s  + 19*s + 2  7*s  + 19*s + 2]{t}
+
+
+
+Example 5
+---------
+
+        .. image:: https://user-images.githubusercontent.com/53227127/133931743-550bfbd7-ef6a-47e7-9661-2f6b70959815.png
+
+Given,
+
+    $G1 = \frac{1}{10 + s}$
+
+    $G2 = \frac{1}{1 + s}$
+
+    $G3 = \frac{1 + s^2}{4 + 4s + s^2}$
+
+    $G4 = \frac{1 + s}{6 + s}$
+
+    $H1 = \frac{1 + s}{2 + s}$
+
+    $H2 = \frac{2 \cdot (6 + s)}{1 + s}$
+
+    $H3 = 1$
```

Comment:
The math directive gives you the latex align environment so you can write and format more nicely with:

```rst
... math::

   a & = b \\
   c & = d
```

Reply:
Now the rendering looks clearer. Thanks !